visual-tests/src/test-fixtures/series/line.js

```diff
@@ -1,12 +1,9 @@
 (function(d3, fc) {
     'use strict';
 
-    var data = fc.utilities.dataGenerator().startDate(new Date(2014, 1, 1))(50);
-
-    var chart = d3.select('#line'),
-        chartLayout = fc.utilities.chartLayout();
-
-    chart.call(chartLayout);
+    var dataGenerator = fc.utilities.dataGenerator()
```

Comment:
I think `fc.charts.linearTimeSeries` should have its own visual test, rather than adapting the current line series test.